src/streamlink/plugins/youtube.py

```diff
@@ -325,16 +288,16 @@ def _get_stream_info(self, video_id):
 
             res = self.session.http.get(self._video_info_url, params=params)
             info_parsed = parse_query(res.content if is_py2 else res.text, name="config", schema=_config_schema)
-            if (info_parsed.get("player_response", {}).get("playabilityStatus", {}).get("status") != "OK"
-                    or info_parsed.get("status") == "fail"):
-                reason = (info_parsed.get("player_response", {}).get("playabilityStatus", {}).get("reason")
-                          or info_parsed.get("reason"))
+            player_response = info_parsed.get("player_response")
+            playability_status = player_response.get("playabilityStatus")
```

Comment:
```
    playability_status = player_response.get("playabilityStatus")
AttributeError: 'NoneType' object has no attribute 'get'
```

if you want to do it in two lines do it like this

```suggestion
            player_response = info_parsed.get("player_response", {})
            playability_status = player_response.get("playabilityStatus", {})
```